src/Command/GraphQL/RebuildDefinitionsCommand.php

```diff
@@ -33,10 +33,10 @@ protected function configure()
             ->setName('datahub:graphql:rebuild-definitions')
             ->setDescription('Rebuild GraphQL endpoint definitions')
             ->addOption(
-            'definitions',
+                'definitions',
                 null,
                 InputOption::VALUE_REQUIRED,
-            'Comma separated list of endpoints'
+                'Comma separated list of endpoints'
```

Comment:
why ?

Reply:
I might have merged 1.0 in my branch, these are not my changes :(